docs/collator-instructions.md

```diff
@@ -0,0 +1,222 @@
+## Set session keys
+
+Now we need to rotate keys and set our keys on chain to associate our on-chain acct with the collator node software in order to join the active set of collators and receive rewards. Ensure the collator is running or this step won't work.
+
+```shell
+sudo curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "author_rotateKeys", "params":[]}' http://127.0.0.1:9933/
+```
+
+You will be greeted with an output that looks like:
+
+```shell
+{"jsonrpc":"2.0","result":"0xprivate_key_will_be_here0","id":1}
+```
+
+"result":"**0x_private_key_will_be_here0**" is what we are interested in.
+
+You need to make sure that you have a Polkadot/Substrate account set up, here's some videos in case you don't know how to do that:
+
+1. Polkadot JS Videos  https://www.youtube.com/watch?v=dG0DP9vayPY    https://www.youtube.com/watch?v=BpTQBAyFvEk
+
+2. Talisman Video   https://docs.talisman.xyz/talisman/talisman-initiation/setup-a-talisman-wallet  
+
+Now that you have made an account using one of those extensions, head on over to the InvArch Tinkernet section of Polkadot JS:
+
+https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftinker.invarch.network#/extrinsics
+
+1. Ensure that you are in the Developer tab (the top header), under Extrinsics->Submission.
+2. In the `using the selected account` field, select the account you just made for the collator.
+3. In the `submit the following extrinsic` field, select `session`.
+4. In the next field (to the right), select `setKeys(keys, proof)`.
+5. In the `keys:` field, paste in your **0x_private_key_will_be_here0** from your node.
+6. In the `proof` field, type in `0`.
+7. Submit the transaction.
+
+
+## Register as a collator candidate (to join the active set)
+### Get the current candidate count
+
+1. Visit the Regular UI: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftinker.invarch.network#/js
+2. Add this script and run it:
+```jsx
+// Simple script to get candidate pool size
+const candidatePool = await api.query.parachainStaking.candidatePool();
+console.log(`Candidate pool size is: ${candidatePool.length}`);
+```
+3. Write down the result
+
+### Join Candidates
+1. Return to the Extrinics page: https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ftinker.invarch.network#/extrinsics
+2. In the `using the selected account` field, select the account you just made for the collator.
+3. In the `submit the following extrinsic` field, select `collatorSelection`.
+4. In the next field (to the right), select `registerAsCandidate()`.
+5. Submit the transaction.
+6. Wait 2 rounds (roughly 12 hours) for your collator registration to take effect.
+
+Congratulations, you should now be onboarded as a collator.
+
+
+## Prologue: Node Monitoring
+
+As a collator node operator, you should also set up monitoring for your node, including Prometheus, Grafana, and AlertManager. Good instructions for that setup can already be found at:
+Prometheus/Grafana (HDX): https://docs.hydradx.io/node_monitoring
```

Comment:
hope it's ok to reference other chains. Better to not duplicate those instructions and have to maintain them.

Reply:
InvArch 🤝 HydraDX